cloud/scope/powervs_cluster.go

```diff
@@ -868,10 +891,20 @@ func (s *PowerVSClusterScope) ReconcileNetwork() (bool, error) {
 	if networkID != nil {
 		s.V(3).Info("Found PowerVS network in IBM Cloud", "networkID", networkID)
 		s.SetStatus(infrav1beta2.ResourceTypeNetwork, infrav1beta2.ResourceReference{ID: networkID, ControllerCreated: ptr.To(false)})
+	}
+	dhcpServerID, err := s.checkDHCPServer()
+	if err != nil {
+		return false, err
+	}
+	if dhcpServerID != nil {
+		s.V(3).Info("Found DHCP server in IBM Cloud", "DHCP Server ID", dhcpServerID)
```

Comment:
To keep uniform and improve our [logging](https://cluster-api.sigs.k8s.io/developer/logging#keyvalue-pairs), lets use , `dhcpServerID` instead of `"DHCP Server ID"`